### ISSUE TITLE
OOPS andes_queue_len 4.8.11-1-ARCH

### ISSUE THREAD

**Issue by SinBirb** [ulli-kroll/mt7610u#7]
Hi,
I have never gotten any driver for my dodocool AC600 Wifi USB dongle to work, so I'm happy to help.

My system always freezes almost instantly after inserting the USB stick. Log: 
[OOPS_andes_queue_len.txt](https://github.com/ulli-kroll/mt7610u/files/620411/OOPS_andes_queue_len.txt).

I think I fixed this by re-inserting the function you deleted in this commit: https://github.com/ulli-kroll/mt7610u/commit/07f560116d0e8d0f59e02108582332e23736849a , it seems to be called by the firmware.

Thanks for your work.

**Comment by ulli-kroll** [comment]
Hi

please post output before you patched this issue ...
Attached post is with removed "andes_queue_len()"

**Comment by SinBirb** [comment]
Yes it's removed because you removed it. This is the log with your code only

**Comment by ulli-kroll** [comment]
I'm lazy copy & paste from 
#8 
Please check current git here.
They was some locking errors in mcu/mcu_and.c
Code is cleaned up but still messy here ...
